File: src/main/java/com/example/model/ConvocatoryInfo.java
package com.example.model;

public class ConvocatoryInfo {
    private String classroom;
    private String date;
    private String convocatory;

    public ConvocatoryInfo(String classroom, String date, String convocatory) {
        this.classroom = classroom;
        this.date = date;
        this.convocatory = convocatory;

    }

    public String getClassroom() {
        return classroom;
    }

    public String getDate() {
        return date;
    }

    public String getConvocatory() {
        return convocatory;
    }

    

}

File: MyURJC/src/app/services/UserService/ConvocatoryInfo.ts
export class ConvocatoryInfo {


    constructor(
        private convocatoryId: number,
        private date: string,
        private convocatory: number,
        private classroom: string
    ) {}

    // Getter para convocatoryId
    public getConvocatoryId(): number {
        return this.convocatoryId;
    }

    // Getter para date
    public getDate(): string {
        return this.date;
    }

    // Getter para convocatory
    public getConvocatory(): number {
        return this.convocatory;
    }

    // Getter para classroom
    public getClassroom(): string {
        return this.classroom;
    }

    

        


}